packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx

```diff
@@ -126,15 +127,44 @@ const OnfidoSdkViewContainer = observer(
                         ],
                     });
                     setIsOnfidoInitialized(true);
+                    setIsOnfidoLoaded(true);
                 } catch (err) {
                     setAPIError(err?.message ?? err);
+                    setIsOnfidoLoaded(true);
+
```

Comment:
can't we set setIsOnfidoLoaded to true in finally block?

Reply:
Thanks for the suggestion @amina-deriv 
Will check on this